dask/dataframe/tests/test_groupby.py

```diff
@@ -74,8 +90,9 @@ def groupby_internal_repr():
     assert isinstance(dp, dd.groupby.SeriesGroupBy)
     assert isinstance(dp._meta, pd.core.groupby.SeriesGroupBy)
     # slicing should not affect to internal
-    assert isinstance(dp.obj, dd.Series)
-    assert_eq(dp.obj, gp.obj)
+    # these asserts currently fails. We're uncertain how to fix this.
+    # assert isinstance(dp.obj, dd.Series)
```

Comment:
dd.Series -> dd.DataFrame.

Reply:
Yes, as you explained below. Also, as you said it probably doesn't matter much. But I'd keep as an xfail because of this inconsistency.